fluidkeys/keyrotate.go

```diff
@@ -23,3 +215,28 @@ func printCheckboxFailure(actionText string, err error) {
 	fmt.Printf("\r        %s\n", colour.Error(fmt.Sprintf("%s", err)))
 }
 
+// printKeyHeaderAndActions outputs a list of actions like this:
+//
+// Rotate foo@example.com:
+//
+//   [ ] Shorten the primary key expiry to 31 Oct 18
+//   [ ] Expire the encryption subkey now (ID: 0xC52C5BD9719C9F00)
+//   [ ] Create a new encryption subkey valid until 31 Oct 18
+
+func printKeyHeaderAndActions(key *pgpkey.PgpKey, actions []status.KeyAction) {
+	if len(actions) == 0 {
+		return
+	}
+
+	fmt.Printf("\nRotate %s:\n\n", colour.Info(displayName(key)))
+
+	for _, action := range actions {
+		fmt.Printf("    [ ] %s\n", action)
+	}
+}
+
+func moveCursorUpLines(numLines int) {
```

Comment:
Is this function redundant?

Reply:
It *was*, but I've added commit https://github.com/fluidkeys/fluidkeys/pull/110/commits/429546895c3cc7b42ee6f50c16b0f09962971d17 which uses it instead of "\z"